packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -217,7 +217,7 @@ export abstract class FluidDataStoreContext extends TypedEventEmitter<IFluidData
         // Thus having slashes in types almost guarantees trouble down the road!
         assert(id.indexOf("/") === -1, 0x13a /* `Data store ID contains slash: ${id}` */);
 
-        this._attachState = this.containerRuntime.attachState !== AttachState.Detached && existing ?
+        this._attachState = this.containerRuntime.attachState !== AttachState.Detached && this.existing ?
```

Comment:
can this.existing be private?

Reply:
Not until we make the existing parameter mandatory inside `IFluidDataStoreFactory.instantiateDataStore`..